### DISCUSSION TITLE
Connecting a groupoffice calendar

### DISCUSSION THREAD

**Discussion by lukaszhandy** [Intermesh/groupoffice#918] [General]
Hi
I have 2 groupoffice instances.
I would like to connect the calendars from the first instance to the second.
Where can I enter the URL of the calendar so that the client instance can connect the calendar from the server instance in full synchronization?
Regards

**Comment by mschering** [comment]
Sorry, that's not supported. GO has a caldav server built in but no client (yet)...

**Comment by thomasgg23** [comment]
Have the same constellation, would love to see a caldav client too.
